subsys/infinispan/src/main/java/org/commonjava/indy/subsys/infinispan/CacheProducer.java

```diff
@@ -90,40 +100,17 @@ public void start()
         }
         else
         {
-            try (InputStream stream = Thread.currentThread().getContextClassLoader().getResourceAsStream( ISPN_XML ))
+            try
             {
-                configuration = IOUtils.toString( stream );
+                logger.info( "Using Infinispan configuration:\n\n{}\n\n", resourceStr );
+                cacheManager = new DefaultCacheManager(
+                        new ByteArrayInputStream( resourceStr.getBytes( StandardCharsets.UTF_8 ) ) );
             }
             catch ( IOException e )
             {
-                throw new RuntimeException( "Cannot read infinispan configuration from classpath: " + ISPN_XML, e );
+                throw new RuntimeException( "Cannot read infinispan configuration.", e );
```

Comment:
Giving more information about which configuration location failed would be helpful if we ever have to troubleshoot one of these errors...